django_basecrm/apps.py

```diff
@@ -51,5 +49,4 @@ def instantiate_stages(self, force=False):
 
     def instantiate_users(self, force=False):
         if force is True or self.users is None:
-            self.users = get_users_from_api()
-
+            self.users = get_users_from_api(**settings.BASECRM_CACHE_USERS_FILTERS)
```

Comment:
Final change is here - we instantiate users using some sensible defaults - 100 users, 'active' only, rather than the current 25 users, 'active' and 'inactive'.